recipes/libmysqlclient/all/conanfile.py

```diff
@@ -109,8 +112,7 @@ def _cmake_new_enough(self, required_version):
 
     def build_requirements(self):
         if is_apple_os(self) and not self._cmake_new_enough("3.18"):
-            # CMake 3.18 or higher is required if Apple, but CI of CCI may run CMake 3.15
-            self.tool_requires("cmake/3.25.3")
+            self.tool_requires("cmake/[^3.18]")
```

Comment:
Can you fix up this range too please? :)

Reply:
Sorry I don't understand. It's fixed already.

Reply:
@RubenRBS I guess you meant that there should be an upper bound? It's fixed now.

Reply:
Yeah sorry, I should have made myself more clear!